content/en/case-studies/peardeck/index.html

```diff
@@ -1,87 +0,0 @@
----
```

Comment:
Could you please specify the reason for removing this file?

Reply:
someone in the community told me that this file is a clone of other issue and that issue is resolved that's why i removed it. Should i undo this file also?

Reply:
sorry it's clear to me. should i again add this file or i just leave it as it is? Could you please specify.

Reply:
Yes, Please add this file. If any further confusion ping me on slack( mk46 )

Reply:
hey i am unable to add the file again and it seems someone has made the changes to that file. what should i do now?

Reply:
I think you missed https://kubernetes.io/docs/contribute/new-content/open-a-pr/#create-a-local-clone-and-set-the-upstream. PTAL.